pkg/storage/stores/tsdb/head_manager.go

```diff
@@ -170,11 +170,9 @@ func (m *HeadManager) loop() {
 	for {
 		select {
 		case <-ticker.C:
-			m.metrics.tsdbHeadRotationsTotal.Inc()
-
 			// retry tsdb build failures from previous run
 			if err := buildPrev(); err != nil {
-				m.metrics.tsdbHeadRotationsFailedTotal.Inc()
+				m.metrics.headRotations.WithLabelValues(statusFailure).Inc()
```

Comment:
I think we should drop this since it can confuse people. We are anyways going to monitor failures in building the tsdb.

Reply:
yeah, idea was to flag this branch for blocking rotations, but i think you are right